modules/ROOT/pages/proc_uninstalling-pipelines-operator.adoc

```diff
@@ -19,22 +21,35 @@
 . Click **Delete**.
 
 . The project is now deleted from the **Projects** list and cluster.
+////
 
+To uninstall pipelines you need to first delete the pipeline configuration custom resource and then uninstall the pipeline operator. Deleting the Pipelines operator will not delete all the custom resources created by default when the operator is installed.
 
-
-== Uninstalling the pipelines operator
+== Uninstalling the pipeline configuration custom resource
 
 [discrete]
-=== Procedure
+.Procedure
+
+Uninstall the custom resource, created by default when you install the Pipelines Operator, as follows:
 
-. Go to **Catalog > OperatorHub** in the Web Console.
+. In the *Administrator* perspective of the Web Console, navigate to *Operators > Installed Operators*.
+. In the *Project* drop-down select the *OpenShift- Operators* project.
+. Click on the *OpenShift Pipelines Operator* to see the overview of the operator.
+. Click on the *OpenShift Pipelines config* tab to see the *Configs* screen.
+. Click the *Options* menu {kebab} adjacent to the *cluster* config listed in the *Configs* screen, and select *Delete Config*.
+. Click *Delete* to confirm the deletion of the custom resource.
```

Comment:
we can also add a warning here:
deleting the "cluster" (instance of config) will delete openshift-pipelines components and all Tasks, Pipelines etc on the cluster will be lost